.travis.yml

```diff
@@ -195,15 +196,13 @@ after_success:
       fi
   - |
     set -e
-    fail=0
     mkdir -p .ci-temp
     if [ -z "$(ls -A .ci-temp)" ]; then
         echo "Folder .ci-temp/ is empty."
     else
         echo "Folder .ci-temp/ is not empty. Verification failed."
         echo "Contents of .ci-temp/:"
-        fail=1
+        ls -A .ci-temp
+        sleep 5s
```

Comment:
if this sleep isn't on the outside, won't we not see the message about ci-temp being empty for the if portion?

Reply:
We might, let me restore it.
This arch might behave a bit different, but let's try

Reply:
last line was printed into log, https://app.travis-ci.com/github/checkstyle/checkstyle/jobs/546277507#L686

but I restored "sleep" just in case.